test/e2e/tests/test_scenarios.py

```diff
@@ -121,6 +121,14 @@ class UserHelper(helper.ResourceHelper):
     Overrides methods as required for custom resources.
     """
 
+    def assert_extra_items_k8s(self, expected_name, expected_value, k8s_resource) -> bool:
+        # events are processed by custom logic
+        # returns AssertError if events are empty in k8s resource
+        if expected_name == "events":
+            assert k8s_resource.get(expected_name) is not None
+            return True
```

Comment:
This condition can be added to `ResourceHelper` so it's not duplicated between all of the classes?

Reply:
It is only for `cluster`, `acl`, and `user` resources. MemoryDB service doesn't generate `events` for other three resources.

Reply:
Oh. Sure.